dist/tools/functions.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# run arguments as command, only print output when command errors
+quiet() {
+    OUT="$("$@" 2>&1)"
+    RES=$?
+    [ -n "$OUT" ] && [ $RES -ne 0 -o "$QUIET" = "0" ] && echo "$OUT"
```

Comment:
Using ```-eq``` fails if not both arguments are an integer. and as QUIET might be use supplied (e.g., QUIET=true), it is safer to to string comparison here.

See ```# test a -eq 0``` -> ```-bash: test: a: integer expression expected```

RES on the other hand is definitely filed with an integer value, so using ```-ne``` is fine.